### ISSUE TITLE
"R2 get started guide" has missing steps

### ISSUE THREAD

**Issue by catharsis71** [cloudflare/cloudflare-docs#4698]
### Which Cloudflare product does this pertain to?

R2

### Existing documentation URL(s)

https://developers.cloudflare.com/r2/get-started/

### Section that requires update

5. Access your R2 bucket from your Worker

​7. Deploy your bucket

### What needs to change?

when following document as written, multiple issues are encountered

Section 5 states "Add the following snippet into your project’s index.js file" however no index.js will exist at this point

Section 7 states to run "wrangler publish" however this will fail due to no package.json

```
$ wrangler publish
✨  Basic JavaScript project found. Skipping unnecessary build!
Error: Your JavaScript project is missing a `package.json` file; is `/home/cmcphers/wrangler` the wrong directory?
```

### How should it change?

There appears to be a missing step, running "wrangler generate" which creates the missing index.js and package.json files, and then cd into the new subdirectory that was created

Even doing that the publish still fails with an error which I haven't yet found a solution for

```
$ wrangler publish
✨  Basic JavaScript project found. Skipping unnecessary build!
Error: Something went wrong with the request to Cloudflare...
Uncaught SyntaxError: Unexpected token 'export'
  at line 2
 [API code: 10021]
```

**Comment by KianNH** [comment]
What version of wrangler are you using?

Generate is deprecated in 2.0 and init will create the package.json for you as well.

**Comment by catharsis71** [comment]
Turns out I was using 1.X

There was something in the doc somewhere saying it could be installed with Cargo instead of NPM, I didn't have NPM installed at the time so I did that but apparently Cargo installs 1.X

**Comment by KianNH** [comment]
The old wrangler was written in Rust and therefore available in Cargo - the new 2.0 is fully TypeScript/Node so only available in NPM.
